### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by bewiwi**

### CODE REVIEW

File: examples/private-cluster-cl/output.tf [hunk 1]
```diff
@@ -30,7 +30,7 @@ Your kubernetes cluster is up.
 
 You can connect in one of the instances:
 
-    $ ssh -J core@${module.network.bastion_public_ip} core@${module.k8s.private_ipv4_addrs[0]}
+    ${join( "    ",formatlist("$ ssh -J core@${module.network.bastion_public_ip} core@%s\n", module.k8s.public_ipv4_addrs))}
```
Comment (yanndegat) [inline examples/private-cluster-cl/output.tf]: you should prefer:
${indent(6, join("\n", list)))}

File: examples/public-cluster-cl/output.tf [hunk 2]
```diff
@@ -28,7 +28,7 @@ Your kubernetes cluster is up.
 
 You can connect in one of the instances:
 
-    $ ssh core@${module.k8s.public_ipv4_addrs[0]}
+    ${join( "    ",formatlist("$ ssh core@%s\n", module.k8s.public_ipv4_addrs))}
```
Comment (yanndegat) [inline examples/public-cluster-cl/output.tf]: idem

**Comment by bewiwi** [comment]
Updated

**Comment by yanndegat** [comment]
great

**Pull request merged**